robotiq_driver/tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Add support for GTest.
+find_package(ament_cmake_gtest REQUIRED)
+find_package(ament_cmake_gmock REQUIRED)
+find_package(ament_lint_auto REQUIRED)
+find_package(ros2_control_test_assets REQUIRED)
+
+ament_lint_auto_find_test_dependencies()
+
+# GMock throws an error if we don't switch off this option in tests.
+if(CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)")
+    add_compile_options(-Wno-sign-conversion)
+endif()
+
+###############################################################################
+# test_robotiq_gripper_hardware_interface
+
+ament_add_gmock(test_robotiq_gripper_hardware_interface
+    test_robotiq_gripper_hardware_interface.cpp
```

Comment:
robotiq_driver/tests/test_robotiq_gripper_hardware_interface.cpp includes ros2_control_test_assetss but you haven't added that include directory anywhere.

I think you're missing an ament_target_dependencies or something.

Reply:
In Humble, this includes comes with the package `ros2_control_test_assets`. But it seems that is not true for iron or rolling.

Reply:
OK, I've fixed it. It seems that Humble was somehow forgiving, but Iron and Rolling required this dependency.

```
ament_target_dependencies(test_epick_gripper_hardware_interface
    ros2_control_test_assets
)
```